src/Workspaces/CSharp/Portable/Simplification/Reducers/CSharpNameReducer.Rewriter.cs

```diff
@@ -152,7 +152,10 @@ public override SyntaxNode VisitArrayType(ArrayTypeSyntax node)
                     this.alwaysSimplify = node.HasAnnotation(Simplifier.Annotation);
                 }
 
-                var result = base.VisitArrayType(node);
+                var result = SimplifyExpression(
+                    node,
+                    newNode: base.VisitArrayType(node),
+                    simplifier: s_simplifyName);
```

Comment:
these were missing.  So the simplifier wouldn't simplify something like "int[]" to "var".

Reply:
Do we have a unit test for this case?

Reply:
yes.  that's how i found this.  :)